### PR TITLE
Transfer MapInt, MapString, MapDict Docs

### PR DESCRIPTION
**Pull request by sync**
see https://github.com/reason-association/reasonml.org/issues/36#issuecomment-538132221

## TODO
- [x] MapInt
- [x] MapString
- [x] MapDict

### CODE REVIEW

File: pages/belt_docs/map.mdx [hunk 1]
```diff
@@ -107,7 +107,7 @@ let findFirstByU: (t('k, 'v, 'id), [@bs] (('k, 'v) => bool)) => option(('k, 'v))
 let findFirstBy: (t('k, 'v, 'id), ('k, 'v) => bool) => option(('k, 'v));
 ```
 
-`findFirstBy(m, p)` uses funcion `f` to find the first key value pair to match predicate `p`.
+`findFirstBy(m, p)` uses function `f` to find the first key value pair to match predicate `p`.
```
Comment (sync) [inline pages/belt_docs/map.mdx]: found a spelling error

File: pages/belt_docs/map-dict.mdx [hunk 2]
```diff
@@ -1,272 +1,306 @@
...
+
+This module seprate identity from data, it is a bit more verboe but slightly more efficient due to the fact that there is no need to pack identity and data back after each operation
+
+***Advanced usage only***
+
+</ApiIntro>
+
 ## t
 
-```re sig
```
Comment (nikgraf) [inline pages/belt_docs/map-dict.mdx]: Why remove the `re sig`?

We plan to use it to apply special styling in the future.

File: pages/belt_docs/map.mdx (hunk 1 above)
Comment (nikgraf) [reply pages/belt_docs/map.mdx]: nice catch, thanks

File: pages/belt_docs/map-dict.mdx (hunk 2 above)
Comment (sync) [reply pages/belt_docs/map-dict.mdx]: oh i looked at the list and none of them had it

**Pull request merged**